packages/nimble-components/src/card/specs/README.md

```diff
@@ -0,0 +1,111 @@
+# Nimble Card
+
+## Overview
+
+The `nimble-card` is a container that is designed to contain arbitrary content specified by a client
+application. The `nimble-card` is intended for grouping related content.
+
+### Background
+
+[GitHub Issue #296: nimble-card component](https://github.com/ni/nimble/issues/296)
+
+An IxD and Visual Design spec for the `nimble-card` component have not yet been provided. We will add some
+preliminary styling in the interim.
+
+The new Routines UX in SystemLink Enterprise calls for grouping related configuration in a slide-out in labeled
+containers. See [Figma: NorthStar Vision for SLE Routines](https://www.figma.com/file/VQ7WIi3qqRG3r19VXqVvem/Stratus-Routines?type=design&node-id=1%3A74129&mode=design&t=KfPTwy4IhHbt42LR-1).
+We are designing the `nimble-card` component to address that use case.
+
+---
+
+## Design
+
+The `nimble-card` will not include any built-in content (title, footer, actions, etc.). Some component libraries
+do provide a card component with built-in, pre-styled content or pre-styled child components to use inside the card
+component. See the [Angular Material `mat-card` component](https://v5.material.angular.io/components/card/overview)
+for an example. We could add similar content in the future, but it is not required at this time.
+
+Since we're not providing any built-in content, the API surface area for the `nimble-card` is very small. The template
+contains only a slot, and there are no configurable properties on the component. Here is an example usage for
+the `nimble-card` component:
+
+```
+<nimble-card>
+    <div>
+        <h2>Heading</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        <nimble-button>Button</$nimble-button>
+    </div>
+</nimble-card>
+```
+
+While we don't have a finalized visual design spec, we expect the style will include a background color and a
+border, potentially with rounded corners. We will add design tokens for the background and border. The GitHub
```

Comment:
> We will add design tokens for the background and border.

Unless we know that the background and border will require different design tokens than our existing nimble components, we should hold off on adding new tokens.

Reply:
That's fair. Can you tell me more about when it is appropriate to create design tokens? It wasn't clear to me from the documentation.

Reply:
We create component-specific design tokens when a given "global" token doesn't apply. For example, we have tokens such as `ni-nimble-border-hover-color` that applies to a number of different controls. So, in that case we don't have tokens for `ni-nimble-button-border-hover-color`, `ni-nimble-select-border-hover-color`, `ni-nimble-combobox-border-hover-color`, etc.

In this case, I'm wondering whether or not our existing `ni-nimble-section-background-color` makes sense for this card (or possibly the card is transparent with just a border indicating the grouping). This may or may not require a card-specific token, but that'll be based on a visual designer telling us that this will be a different pattern than other components. Same for the border color -- we may use an existing token like `ni-nimble-divider-background-color`, or it may be a new color.

Reply:
Thanks for the information, that makes sense. We'll wait for visual design input before adding new tokens. (The prototype branch has tokens for background and border, but that was just so I could learn how they work.)

I added a sentence about not creating tokens yet.